src/Algebra/Properties/Ring.agda

```diff
@@ -30,3 +31,24 @@ open RingWithoutOneProperties ringWithoutOne public
   - 1# * x    ≈⟨ -‿distribˡ-* 1# x ⟨
   - (1# * x)  ≈⟨ -‿cong ( *-identityˡ x ) ⟩
   - x         ∎
+
+------------------------------------------------------------------------
+-- Reasoning combinators inherited from Monoid
+
+open PM +-monoid using () renaming
+  ( ε-unique to 0#-unique; ε-comm to 0#-comm
+  ; elimʳ to +-elimʳ; introʳ to +-introʳ
+  ; elimˡ to +-elimˡ; introˡ to +-introˡ
+  ; introᶜ to +-introᶜ
+  ; cancelʳ to +-cancel-invʳ; insertʳ to +-insertʳ
+  ; cancelˡ to +-cancel-invˡ; insertˡ to +-insertˡ
+  ; cancelᶜ to +-cancel-invᶜ; insertᶜ to +-insertᶜ) public
```

Comment:
My only nitpick here, in terms of `style-guide` (L190) is as follows:
```suggestion
open PM +-monoid public
   using ()
   renaming ( ε-unique to 0#-unique; ε-comm to 0#-comm
  ; elimʳ to +-elimʳ; introʳ to +-introʳ
  ; elimˡ to +-elimˡ; introˡ to +-introˡ
  ; introᶜ to +-introᶜ
  ; cancelʳ to +-cancel-invʳ; insertʳ to +-insertʳ
  ; cancelˡ to +-cancel-invˡ; insertˡ to +-insertˡ
  ; cancelᶜ to +-cancel-invᶜ; insertᶜ to +-insertᶜ)
```
plus or minus some additional offset whitespace.